examples/tests/error/inflight_stacktraces.test.w

```diff
@@ -12,7 +12,7 @@ test "expect.equal" {
 }
 
 test "bucket failed get" {
-  bucket.get("doesn't exist");
+  bucket.delete("doesn't exist", mustExist: true);
```

Comment:
Why change this particular test?

If it is needed, the test name should reflect it

Reply:
The error raised by `bucket.get()` in the simulator includes an absolute file system path where we the simulator tried to read a file from, which was resulting in a non-deterministic test snapshot. Just being a little lazy... I'll fix the test name. :)